spec/controllers/api/encryptables_controller_spec.rb

```diff
@@ -15,13 +15,15 @@
   let(:credentials1) { encryptables(:credentials1) }
   let(:encryptable) { encryptables.first }
 
+
```

Comment:
Remove that one unnecessary line, yes this is nitpicky but hey there was nothing else to complain about :joy: